packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -113,6 +113,9 @@ class _DefaultDoctorValidatorsProvider implements DoctorValidatorsProvider {
       if (windowsWorkflow.appliesToHostPlatform)
         _workflows.add(windowsWorkflow);
 
+      if (webWorkflow.appliesToHostPlatform)
```

Comment:
(optional, nit) switch to new syntax.
```dart
return _workflows ??= <Workflow>[
  if (iosWorkflow.appliesToHostPlatform) iosWorkflow,
  if (androidWorkflow.appliesToHostPlatform) androidWorkflow,
  ...
];
```

Reply:
Worth doing, but there are some timing constraints with this fix...